### ISSUE TITLE
Can you teach me how to install with npm after downloading this file.

### ISSUE THREAD

**Issue by Oluwasetemi** [joshbuchea/atom-html-head-snippets#3]
correct the installation in the readme to npm. is there anything like apm?

**Comment by joshbuchea** [comment]
@Oluwasetemi I'm not sure I understand your question. Could you rephrase your question please?

**Comment by Oluwasetemi** [comment]
1. I don't know how to install with npm after cloning a repo.
2. change the text of the installation in the readme to "npm" and not "apm"

**Comment by joshbuchea** [comment]
`apm` is a thing. Atom package manager. [apm](https://github.com/atom/apm)

The README contains the correct steps to install the project.

**Comment by Oluwasetemi** [comment]
okay
